WordPress/src/main/java/org/wordpress/android/push/GCMRegistrationIntentService.java

```diff
@@ -80,8 +81,9 @@ public void sendRegistrationToken(String gcmToken) {
             }
 
             // Register to other kind of notifications
-            // TODO: Handle Zendesk PNs instead
-//            HelpshiftHelper.getInstance().registerDeviceToken(this, gcmToken);
+            if (mAccountStore.hasAccessToken()) {
+                ZendeskHelper.enablePushNotifications();
```

Comment:
this can probably be moved this call under the previous `if ` clause above, preceded by the clarifying ` // Register to other kind of notifications` comment as well

Reply:
This is kind of a dirty implementation for now, I am not even sure if this code will stay here, but you're right.